CHANGELOG.md

```diff
@@ -1,3 +1,11 @@
+# dbt_google_ads v0.9.1
+## Bug fixes
+[PR #41](https://github.com/fivetran/dbt_google_ads/pull/41) includes the following bug fixes:
+- Adjusted keyword report to leverage the stats ids as opposed to the history ids to have more accurate reporting.
```

Comment:
Since there is only one change we can condense the PR reference to the entry.
```suggestion
- Adjusted keyword report to leverage the stats ids as opposed to the history ids to have more accurate reporting. ([PR #41](https://github.com/fivetran/dbt_google_ads/pull/41))
```